data/core/terrain.cfg

```diff
@@ -1377,7 +1377,7 @@ Occasionally caves are <italic>text='illuminated'</italic>."
     id=cavewall_earthy_hewn
     name= _ "Cave Wall"
 #ifndef SCHEMA_VALIDATION
-    editor_name="Reinforced Earthy Cave Wall"
+    editor_name= _ "Reinforced Earthy Cave Wall"
```

Comment:
This one was intentionally left non-translatable, as shown by the `#ifndef` above it.